cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,28 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
```

Comment:
Bad practice to visit the page inside the beforeEach hook. It's better to visit the page inside the specific test case because visiting the page might not be required for all the tests in the describe block.